MultiVersions/Forge/main/ecologics/lang/zh_tw.json

```
@@ -0,0 +1,168 @@
+{
+  "block.ecologics.coconut": "椰子",
+  "block.ecologics.hanging_coconut": "懸掛椰子",
+  "block.ecologics.coconut_button": "椰木按鈕",
+  "block.ecologics.coconut_door": "椰木門",
+  "block.ecologics.coconut_fence": "椰木柵欄",
+  "block.ecologics.coconut_fence_gate": "椰木柵欄門",
+  "block.ecologics.coconut_seedling": "椰木幼苗",
+  "block.ecologics.coconut_leaves": "椰木樹葉",
+  "block.ecologics.coconut_log": "椰木原木",
+  "block.ecologics.coconut_planks": "椰木材",
+  "block.ecologics.coconut_pressure_plate": "椰木壓力板",
+  "block.ecologics.coconut_slab": "椰木半磚",
+  "block.ecologics.coconut_stairs": "椰木階梯",
+  "block.ecologics.coconut_trapdoor": "椰木地板門",
+  "block.ecologics.coconut_wood": "椰木塊",
+  "block.ecologics.seashell": "貝殼",
+  "block.ecologics.seashell_block": "貝殼磚",
+  "block.ecologics.seashell_tile_slab": "貝殼磚瓦半磚",
+  "block.ecologics.seashell_tile_stairs": "貝殼磚瓦階梯",
+  "block.ecologics.seashell_tile_wall": "貝殼磚瓦牆",
+  "block.ecologics.seashell_tiles": "貝殼磚瓦",
+  "block.ecologics.sandcastle": "沙堡",
+  "block.ecologics.stripped_coconut_log": "剝皮椰木原木",
+  "block.ecologics.stripped_coconut_wood": "剝皮椰木塊",
+  "block.ecologics.coconut_sign": "椰木告示牌",
+  "block.ecologics.coconut_wall_sign": "牆上的椰木告示牌",
+  "block.ecologics.prickly_pear": "仙人掌果實",
+  "block.ecologics.snow_brick_slab": "雪磚半磚",
+  "block.ecologics.snow_brick_stairs": "雪磚階梯",
+  "block.ecologics.snow_brick_wall": "雪磚牆",
+  "block.ecologics.snow_bricks": "雪磚",
+  "block.ecologics.ice_brick_slab": "冰磚半磚",
+  "block.ecologics.ice_brick_stairs": "冰磚階梯",
+  "block.ecologics.ice_brick_wall": "冰磚牆",
+  "block.ecologics.ice_bricks": "冰磚",
+  "block.ecologics.pot": "壺",
```

我看遊戲中是使用陶土合成，那麼叫做陶壺如何？
```suggestion
  "block.ecologics.pot": "陶壺",
```